curvefs/src/client/fuse_client.cpp

```diff
@@ -537,6 +595,9 @@ CURVEFS_ERROR FuseClient::FuseOpSymlink(fuse_req_t req, const char *link,
 CURVEFS_ERROR FuseClient::FuseOpLink(fuse_req_t req, fuse_ino_t ino,
     fuse_ino_t newparent, const char *newname,
     fuse_entry_param *e) {
+    if (strlen(newname) > option_.maxNameLength) {
```

Comment:
There are many same logics, i think we can define a function to do some precheck, like `max name length` and others.

Reply:
Currently, there are only two line is doing precheck, maybe we can  do reusing  after adding more logical in the future.